src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -170,6 +170,33 @@ function updateFacilitiesSchemaAndData(parents, facilities, schema, data) {
   return { schema: newSchema, data: newData };
 }
 
+function resetFormDataOnTypeOfCareChange(pages, oldData, data) {
```

Comment:
I had moved this logic out of one action and into another, but it actually needs to be in both places, so we properly reset some state when the type of care changes.